container/image_test.py

```diff
@@ -760,6 +760,185 @@ def test_war_image(self):
+        './app/testdata/nodejs_image_binary.runfiles/npm/node_modules/@types/node/path.d.ts', 
+        './app/testdata/nodejs_image_binary.runfiles/npm/node_modules/@types/node/perf_hooks.d.ts', 
+        './app/testdata/nodejs_image_binary.runfiles/npm/node_modules/@types/node/process.d.ts', 
+        './app/testdata/nodejs_image_binary.runfiles/npm/node_modules/@types/node/punycode.d.ts', 
+        './app/testdata/nodejs_image_binary.runfiles/npm/node_modules/@types/node/querystring.d.ts', 
+        './app/testdata/nodejs_image_binary.runfiles/npm/node_modules/@types/node/readline.d.ts', 
+        './app/testdata/nodejs_image_binary.runfiles/npm/node_modules/@types/node/repl.d.ts', 
+        './app/testdata/nodejs_image_binary.runfiles/npm/node_modules/@types/node/stream.d.ts', 
+        './app/testdata/nodejs_image_binary.runfiles/npm/node_modules/@types/node/string_decoder.d.ts', 
+        './app/testdata/nodejs_image_binary.runfiles/npm/node_modules/@types/node/timers.d.ts', 
+        './app/testdata/nodejs_image_binary.runfiles/npm/node_modules/@types/node/tls.d.ts', 
+        './app/testdata/nodejs_image_binary.runfiles/npm/node_modules/@types/node/trace_events.d.ts', 
+        './app/testdata/nodejs_image_binary.runfiles/npm/node_modules/@types/node/ts3.2', 
+        './app/testdata/nodejs_image_binary.runfiles/npm/node_modules/@types/node/ts3.2/globals.d.ts', 
+        './app/testdata/nodejs_image_binary.runfiles/npm/node_modules/@types/node/ts3.2/index.d.ts', 
+        './app/testdata/nodejs_image_binary.runfiles/npm/node_modules/@types/node/ts3.2/util.d.ts', 
+        './app/testdata/nodejs_image_binary.runfiles/npm/node_modules/@types/node/tty.d.ts', 
+        './app/testdata/nodejs_image_binary.runfiles/npm/node_modules/@types/node/url.d.ts', 
+        './app/testdata/nodejs_image_binary.runfiles/npm/node_modules/@types/node/util.d.ts', 
+        './app/testdata/nodejs_image_binary.runfiles/npm/node_modules/@types/node/v8.d.ts', 
+        './app/testdata/nodejs_image_binary.runfiles/npm/node_modules/@types/node/vm.d.ts', 
+        './app/testdata/nodejs_image_binary.runfiles/npm/node_modules/@types/node/worker_threads.d.ts', 
+        './app/testdata/nodejs_image_binary.runfiles/npm/node_modules/@types/node/zlib.d.ts', 
+        '/app', 
+        '/app/testdata', 
+        '/app/testdata/nodejs_image_binary', 
+        '/app/testdata/nodejs_image_binary.runfiles', 
+        '/app/testdata/nodejs_image_binary.runfiles/io_bazel_rules_docker', 
+        '/app/testdata/nodejs_image_binary.runfiles/io_bazel_rules_docker/external'
+      ]
+      self.assertLayerNContains(img, 1, layerOneFiles)
+
+      # Check that the next layer contains node_args
+      layerTwoFiles = [
+        '.',
+        './app',
+        './app/testdata',
+        './app/testdata/nodejs_image_binary.runfiles',
+        './app/testdata/nodejs_image_binary.runfiles/nodejs_darwin_amd64',
```

Comment:
how do we make this test platform agnostic?

Reply:
or should this be platform specific based on the container OS?

Reply:
this should be platform specific based on the container OS used in tests.

Reply:
adjusted it to `nodejs_linux_amd64`